js/src/edit-block/components/top-notice.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { QuestionIcon, TemplateFile } from './';
+
+/**
+ * @typedef {Object} Template The current template.
+ * @property {boolean} templateExists Whether the template was found.
+ * @property {string} templatePath The path of the template.
+ */
+
+/**
+ * @typedef {Object} TopNoticeProps The component props.
+ * @property {boolean} isOnboarding Whether the onboarding should display now.
+ * @property {Template} template The template data.
+ */
+
+/**
+ * The top notice section.
+ *
+ * @param {TopNoticeProps} props
+ * @return {React.ReactElement} The top notice.
+ */
+const TopNotice = ( {
```

Comment:
Here's what this looks like: 

<img width="964" alt="another-b" src="https://user-images.githubusercontent.com/4063887/102920430-62c3be80-4450-11eb-8cc3-dacec58c59ef.png">